spark/src/main/java/org/apache/zeppelin/spark/SparkInterpreter.java

```diff
@@ -1172,6 +1174,14 @@ public InterpreterResult interpret(String line, InterpreterContext context) {
       return new InterpreterResult(Code.ERROR, "Spark " + sparkVersion.toString()
           + " is not supported");
     }
+
+    ObjectMapper om = new ObjectMapper();
+    om.configure(SerializationFeature.FAIL_ON_EMPTY_BEANS, false);
+    Map<String, Map<String, String>> userCredentials = om.convertValue(
+            context.getAuthenticationInfo().getUserCredentials().getUserCredentials(), Map.class);
+    interpreter.intp().directBind("userCredentials",
```

Comment:
But userCredentials is different if you are using different accounts, what can be accessible is just your own private username and password. 
[See here how to create credentials](https://zeppelin.apache.org/docs/0.8.0-SNAPSHOT/security/datasource_authorization.html)